content/blog/structural-alignment-2021.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "FU Softwarepraktikum for 2021: structural alignment"
+date: 2021-01-08T14:27:59+01:00
+---
+
+The new edition of the "Structural Alignment and Superposition" internship is now open for registration!
+
+In this internship, an open-source structural alignment Python package for biopolymers, i.e. proteins and ligands, that we started implementing in 2020 will be further developed and released. While diverse structural alignment implementations can be found in visualization software such as PyMol, VMD or UCSF Chimera, a standalone package is currently missing in the Python ecosystem. We have started filling that gap with a modern Python package designed under the current best practices for development, testing and deployment.
+
+The main focus of this year’s course will be on finalizing the package. This will include, getting familiar with the current code base, working on alignment-specific issues such as automatic structural renumbering as well as benchmarking the tool.
+
+More details:
+
+- Check the [slides](#missing).
```

Comment:
Todo add link once finished

Reply:
pdf from 2020 was added to static/files, but not the updated one?

Reply:
Yes, will upload once we are happy with it!